assets/l10n/app_en.arb

```diff
@@ -160,6 +160,41 @@
       "message": {"type": "String", "example": "Invalid format"}
     }
   },
+  "errorLoadingServerDataShort": "Error loading server data from {url}",
```

Comment:
Let's try this wording:
```suggestion
  "errorLoadingServerDataShort": "Error connecting to Zulip. Retrying…",
```

and for the details:

`Error connecting to Zulip at {serverUrl}. Will retry:\n\n{error}`

Then use that for both the `Server5xxException() || NetworkException()` case and the default case. Both cases are about "an error loading server data", or "an error connecting to Zulip", anyway.